src/scripts/main.js

```diff
@@ -1,3 +1,427 @@
+  excludeIds = [];
+
+  for (let i = min; i <= max; i++) {
+    for (let j = min; j <= max; j++) {
+      const id = i + '' + j;
+
+      if (document.getElementById(id).innerHTML !== '') {
+        moveLeft(id);
+      }
+    }
+  }
+
+  if (isMoved === true) {
+    update();
+  }
+
+  return false;
+}
+
+function moveLeft(id) {
+  if (!id.endsWith(min)) {
+    const arr = id.split('');
+    const i = parseInt(arr[0]);
+    const j = parseInt(arr[1]);
+
+    for (let k = (j - 1); k >= min; k--) {
+      const nId = i + '' + k;
+
+      const val = parseInt(document.getElementById(i + '' + (k + 1)).innerHTML);
+      const nVal = parseInt(document.getElementById(nId).innerHTML);
+
+      if (document.getElementById(nId).innerHTML !== '') {
+        if (val === nVal) {
+          if (excludeIds.indexOf(nId) === -1) {
+            excludeIds.push(nId);
+            document.getElementById(nId).innerHTML = (val + nVal);
+            document.getElementById(nId).classList.replace(`field-cell--${nVal}`, `field-cell--${val + nVal}`);
+            document.getElementById(i + '' + (k + 1)).innerHTML = '';
+            document.getElementById(i + '' + (k + 1)).classList.remove(`field-cell--${val}`);
+            isMoved = true;
+            score += (val + nVal);
+          }
+        }
+        break;
+      } else {
+        document.getElementById(nId).innerHTML = document.getElementById(i + '' + (k + 1)).innerHTML;
+        document.getElementById(nId).classList.add(`field-cell--${val}`);
+        document.getElementById(i + '' + (k + 1)).innerHTML = '';
+        document.getElementById(i + '' + (k + 1)).classList.remove(`field-cell--${val}`);
+        isMoved = true;
+      }
+    }
+  }
+
+  return false;
+}
+
+function down() {
+  isMoved = false;
+  excludeIds = [];
+
+  for (let j = min; j <= max; j++) {
+    for (let i = max; i >= min; i--) {
+      const id = i + '' + j;
+
+      if (document.getElementById(id).innerHTML !== '') {
+        moveDown(id);
+      }
+    }
+  }
+
+  if (isMoved === true) {
+    update();
+  }
+
+  return false;
+}
+
+function moveDown(id) {
+  if (!id.startsWith(max)) {
+    const arr = id.split('');
+    const i = parseInt(arr[0]);
+    const j = parseInt(arr[1]);
+
+    for (let k = (i + 1); k <= max; k++) {
+      const nId = k + '' + j;
+
+      const val = parseInt(document.getElementById((k - 1) + '' + j).innerHTML);
+      const nVal = parseInt(document.getElementById(nId).innerHTML);
+
+      if (document.getElementById(nId).innerHTML !== '') {
+        if (val === nVal) {
+          if (excludeIds.indexOf(nId) === -1) {
+            excludeIds.push(nId);
+            document.getElementById(nId).innerHTML = (val + nVal);
+            document.getElementById(nId).classList.replace(`field-cell--${nVal}`, `field-cell--${val + nVal}`);
+            document.getElementById((k - 1) + '' + j).innerHTML = '';
+            document.getElementById((k - 1) + '' + j).classList.remove(`field-cell--${val}`);
```

Comment:
avoid code duplicate - move to function repeatable lines.  Please improve your code style according this comment not only in these line but on whole your solution